Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+Parser:
+  Title: Palo Alto Network Sessions
+  Version: '0.1'
+  LastUpdated: Dec 20, 2021
+Product:
+  Name: PaloAlto
+Normalization:
+  Schema: NetworkSessions
+  Version: '0.2.0'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This Query Parser maps Palo Alto PanOS Firewall Events (CommonSecurityLogs) to the Advanced SIEM Information Model Network Session schema.
+ParserName: ASimNetworkSessionPaloAltoPanOS
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let NWParser=(disabled:bool=false){
+  CommonSecurityLog | where not(disabled)
+  | where DeviceVendor=="Palo Alto Networks" and (Activity=="TRAFFIC")
+  | parse AdditionalExtensions with "PanOSDstPackets="DstPackets:long";PanOSsrc_packets="SrcPackets:long";start="EventStartTime:datetime";reason="*
+  | project-rename 
+        EventVendor=DeviceVendor // Not documented
```

Comment:
Let' use constant values and decide what they are. We should start a document with those (this is on me)

Reply:
Constant it is